web/samples/samplehdr.js

```diff
@@ -42,7 +42,7 @@
 
   function loadKeyboards() 
   { 
-    var kmw=tavultesoft.keymanweb;
+    var kmw=keyman;
```

Comment:
line 85 will also need to be changed for kmw

Reply:
Thanks, got it now.